notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1276_star_selector_pointing_tests.ipynb

```diff
@@ -0,0 +1,231 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5066b70-de7b-4618-b0f4-746e029f8ebc",
+   "metadata": {},
+   "source": [
+    "## SITCOM-1276: Notebook to find pairs of stars for star-tracker tests with slew and settle¶\n",
+    "\n",
+    "This notebook will produce a list of pairs of stars brighter than Vmag_max, at a given UTC t0, separated by an angular range between sep_lo and sep_hi, and higher than elevation elevation_min.\n",
+    "\n",
+    "This is to produce these positions manually. This notebook will be followed up by script that produces a file with the positions and write a json BLOCK for the corresponding observations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3098bdeb-30c9-4818-86ee-11ea1ca826f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "from astropy.coordinates import AltAz, EarthLocation, SkyCoord\n",
+    "from astropy.time import Time\n",
+    "from astroquery.vizier import Vizier"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6f6a9f9c-50b2-4019-a3cb-813b666763f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define some configuration variables\n",
+    "Vmag_max = 5 # maximum magnitude of the stars in the list\n",
+    "t0 = '2024-3-8 01:00:00' #in UTC, ideally the central time at which observations will take place\n",
```

Comment:
I'm wondering about how we will do this in the block.  I think t0  is only to determine stars that are high enough correct?

I suppose we might need to indicate in the block for what period it is usable.

Maybe we could also ask Tiago if there is some sort of constraint where we give it a list but it only uses the ones that are visible.

Reply:
My view is that this notebook would give you all pairs for a given 'central' UTC time. One would have to give a particular elevation margin to take into account that some stars could be quite low after a certain point. Within a block, I am in the process of figuring this out for the next ticket.